wordpress/var/www/html/wp-cli.yml

```diff
@@ -0,0 +1 @@
+path: wordpress/
```

Comment:
No, not really, the url param is really only used with WP multisite installs to specify which site in the multisite you are running the commands on. It lets you not have to worry about the wp_<siteid>_tablename in the database and perform operations per site, such as 'wp get option' and things like that.